registry/handlers/manifests.go

```diff
@@ -322,7 +333,7 @@ func (imh *manifestHandler) PutManifest(w http.ResponseWriter, r *http.Request)
 		return
 	}
 
-	isAnOCIManifest := mediaType == v1.MediaTypeImageManifest || mediaType == v1.MediaTypeImageIndex
+	isAnOCIManifest := mediaType == v1.MediaTypeImageManifest || mediaType == v1.MediaTypeArtifactManifest || mediaType == v1.MediaTypeImageIndex
```

Comment:
nit: we may refactor it using a switch case.

Reply:
Done.